dynamic_programming/knapsack.py

```diff
@@ -1,6 +1,11 @@
 """
 Given weights and values of n items, put these items in a knapsack of capacity W to get the maximum total value in the knapsack.
+
+Note that only the integer weights 0-1 knapsack problem is solvable using dynamic programming.
 """
+from __future__ import print_function, division, absolute_import
+
+
 def MF_knapsack(i,wt,val,j):
```

Comment:
Please add Python type hints so that https://github.com/python/mypy can test these functions in our automated testing.